gamestonk_terminal/comparison_analysis/yahoo_finance_view.py

```diff
@@ -268,7 +268,9 @@ def correlation(other_args, df_stock, ticker, start, interval, similar):
                 df_stock = d_stock[ticker][
                     d_candle_types[ns_parser.type_candle]
                 ].rename(ticker)
-                for symbol in d_stock.items():
+
+                for symbols in d_stock.items():
```

Comment:
could also be `d_stock.keys()`:)

Reply:
I did that in the first commit and got a bunch of pylint errors so just did this way

Reply:
I was thinking if you needed to wrap the dict.keys() with a list(). Prob, yes.